openfecwebapp/filters.py

```diff
@@ -85,3 +85,7 @@ def get_max(list):
 @app.template_filter()
 def get_min(list):
     return min(list)
+
+@app.template_filter()
+def strip_zero_pad(number):
```

Comment:
Would this template filter be better named as just `strip` if we need to strip other characters than zero padded numbers?